src/commands/createFunction.ts

```diff
@@ -182,4 +184,8 @@ export async function createFunction(
 
     const newFileUri: vscode.Uri = vscode.Uri.file(newFilePath);
     vscode.window.showTextDocument(await vscode.workspace.openTextDocument(newFileUri));
+
+    if (telemetryReporter) {
```

Comment:
Would it be better to hang this onto the main azureFunctions.createFunction/createProject events?  We'd need a way to do that, though, so may not be worth it.

Reply:
Refactored to do just that - let me know what you think

Reply:
Cool.  Are we still guaranteed to receive the event even if you open a new workspace?

Reply:
Not sure. I'll look into that separately: https://github.com/Microsoft/vscode-azurefunctions/issues/93

I don't want to create a separate telemetry event if I don't have to